sh.exe.stackdump

```diff
@@ -0,0 +1,11 @@
+Stack trace:
```

Comment:
Please remove local file.

btw, this means that we need to add one more item to our .gitignore file.

Reply:
Sure